### DISCUSSION TITLE
Code (backticks) in headings

### DISCUSSION THREAD

**Discussion by borekb** [facebook/docusaurus#5065] [Q&A] [answered]
I'd like to have a document with `<code>` in its H1:

```markdown
# The `xyz` command

This page is about the `xyz` command.
```

This renders literally, with backticks, in the sidebar and in the page itself:

<img width="918" alt="Screen Shot 2021-06-25 at 14 40 13" src="https://user-images.githubusercontent.com/101152/123426214-697beb80-d5c3-11eb-871e-c3c832149b47.png">

Updating the Markdown to this didn't help:

```markdown
# The <code>xyz<code> command

This page is about the `xyz` command.
```

<img width="1164" alt="Screen Shot 2021-06-25 at 14 42 07" src="https://user-images.githubusercontent.com/101152/123426392-a7790f80-d5c3-11eb-82ff-588ef2c588a3.png">

Is there a way to do it?

**Comment by slorber** [comment, marked as answer]
It is supposed to work in beta.2, fixed recently

**Reply by borekb** [reply]
Oh, good reason to upgrade! Thanks!

**Reply by slorber** [reply]
example usage found here: 
https://docusaurus.io/docs/docusaurus.config.js
https://redux-toolkit.js.org/api/configureStore

**Reply by borekb** [reply]
Was there a discussion that `<code>` could be rendered in the sidebar as well?

**Reply by slorber** [reply]
We don't support that, and it would be quite complicated to support, considering your page title can actually be a React/MDX component 

Page title, pagination next/previous and sidebar label are all pure text, no markdown/MDX support
